lib/themeprovider.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+// import 'package:riverpod/riverpod.dart';
+
+// final themeprovider = ChangeNotifierProvider((ref) => ThemeSetter());
+
+final themeProvider = ChangeNotifierProvider((ref) => themeNotifier());
+
+class themeNotifier extends ChangeNotifier {
+  ThemeMode _value = ThemeMode.light;
+  ThemeMode get value => _value;
+
+  void setDark() {
+    _value = ThemeMode.dark;
+    notifyListeners();
+  }
+  void setLight() {
+    _value = ThemeMode.light;
+    notifyListeners();
+  }
+  void setSystem() {
+    _value = ThemeMode.system;
+    notifyListeners();
+  }
+}
+
+
+
+// class ThemeSetter extends ChangeNotifier {
```

Comment:
Can you please remove the commented out lines if not required anymore?

If its required, please comment why its required in the code section itself.

Reply:
Ya sure, I forgot to remove it